.github/template-featureRequest.md

```diff
@@ -0,0 +1,18 @@
+**Issue Title:** Feature: [your request in a line] OR Enhancement: [your suggestion in a line]
+
+Copy-and-paste the following content to the issue tracker and fill in with relevant details:
+```markdown
+**Describe the feature/enhancement you would like to see added to TEAMMATES.**
+
+
+
+**How would your proposed feature/enhancement benefit TEAMMATES?**
+
+
+
+**Are there existing similar features? If yes, why do they fall short of your expectations?**
+
+
+
+**Is the proposed feature/enhancement for the benefit of developers or users?**
```

Comment:
`Will the proposed feature/enhancement benefit the developers or users?` sounds better.